README.md

```diff
@@ -3,7 +3,147 @@
 Gradle Plugin based on the phoenixnap/springmvc-raml-plugin:  
 https://github.com/phoenixnap/springmvc-raml-plugin
 
-#Usage
+The plugin is delivered to the gradle plugin repository
+https://plugins.gradle.org/plugin/com.netfalo.raml
+
+
+## Add plugin dependecy
+In the build.gradle add the plugin dependecy
+```
+plugins {
+  id "com.netfalo.raml" version "0.1.0-beta1"
+}
+```
+
+## Tasks
+There are two new tasks added with the plugin
```

Comment:
The new line spaces are missing from the end of this line, so this line appears on the same line as the next one.